build_tools/mako/benchmark_modules_on_android.sh

```diff
@@ -111,10 +111,12 @@ function run_and_log {
     "${DEVICE_ROOT}/iree-benchmark-module" \
     "--flagfile=${DEVICE_ROOT}/flagfile" \
     "--module_file=${DEVICE_ROOT}/${model}-${target}.vmfb" \
-    "--driver=${driver}" | tee "${test_out}"
-  for ms in "$(sed -En -e "${SED_EXPR}" "${test_out}")"; do
+    "--driver=${driver}" \
+    --benchmark_repetitions=10 \
+    --benchmark_min_time=1 | tee "${test_out}"
```

Comment:
Nit relative to the description. This isn't warmup time, it's the sampling time

Reply:
I thought benchmark_min_time is to set the running time before real benchmarks? I tried with fullyconnect test, and the total execution time is longer, but the report time is still less than 1 sec, like 0.107 ms. So I thought this is kind of warmup time.

https://github.com/google/benchmark/blob/bf585a2789e30585b4e3ce6baf11ef2750b54677/src/benchmark.cc#L62-L67

Reply:
No benchmark is already averaging over many many runs for the report time. You should see a number of iterations as part of the report. The way it works is approximately like this. For each benchmark:

1. Run the benchmark a few times to see how long it takes. This also serves as a warmup.
2. Based on how long the benchmark took during these runs, calculate how many times to run it such that it runs at least once, not more than 1e9 times, and for at least `benchmark_min_time`. (I think there is also some observation of the variance on the initial runs the informs the decision here).
3. Run it that many times.
4. Report the average for those iterations as well as the number of iterations.
5. If `benchmark_repetitions` is set, repeat for that many repetitions (I think it may not repeat step 1 though) and report statistical aggregations across those repetitions.

See https://github.com/google/benchmark#runtime-and-reporting-considerations

Anyway, increasing `benchmark_min_time` is fine, but unless you have a reason you think we need to increase it, I would just leave it as the default. It does not increase the warmup time, only the sampling time.

Reply:
I see, thanks for the detailed explanation! Let's leave it as the default. I'll update the commit message.